class BadRequest(Exception):
    pass

class RepoInternalError(Exception):
    pass